src/chrome/content/rules/Criteo.xml

```diff
@@ -19,15 +19,1472 @@ Fetch error: http://pixel.criteo.net/ => https://pixel.criteo.net/: (60, 'SSL ce
 
 <ruleset name="Criteo (partial)" default_off='failed ruleset test'>
```

Comment:
Please note that this rule is still `default_off`.

Please also add `^criteo.com`.